tests/regression/39-signed-overflows/06-abs.c

```diff
@@ -0,0 +1,22 @@
+//PARAM: --enable ana.int.interval --set ana.activated[+] tmpSpecial
```

Comment:
For this to be of any use, we should activate `tmpSpecial` in our svcomp conf?

Reply:
Yes, I think that would need to be activated.

Reply:
Julian has done so in the https://github.com/goblint/analyzer/tree/benchmark_sv_comp branch.

Reply:
Yes, there I added `tmpSpecial` it to the activated analyses in `svcomp24.json`. Since @stilscher added an autotuner, having `tmpSpecialAnalysis` in the autotuner setting should be preferable. 
Here, I therefore added `tmpSpecialAnalysis` to `ana.autotune.activated`.